service/matching/physical_task_queue_manager.go

```diff
@@ -99,6 +102,9 @@ type (
 		taskValidator              taskValidator
 		tasksAddedInIntervals      *taskTracker
 		tasksDispatchedInIntervals *taskTracker
+		// isDeploymentWorkflowStarted keeps track if we have started a deployment workflow for this
+		// physicalTaskQueue
```

Comment:
The following comment captures the intention slightly better. Because what matching really care is registration, workflow creation is just a side effect of the registration.

```suggestion
		// isDeploymentWorkflowStarted keeps track if we have already registered the task queue worker
		// in the deployment.
```